cmd/minikube/cmd/root.go

```diff
@@ -23,6 +23,7 @@ import (
 	"path/filepath"
 	"runtime"
 	"strings"
+	"time"
```

Comment:
lets add integration test for this, we could add a subtest to "TestJSONOutput" that will check if the audit step is logged.

https://github.com/medyagh/minikube/blob/dce38fb331b6172c0c618ad486ac1b60661af687/test/integration/json_output_test.go#L55

Reply:
`audit.Log` doesn't output the event to stdout. We talked about how that might possibly interfere with 3rd party tools that are using the output to keep track of steps, so I excluded it from the json output. Would it make more sense to output it here or leave it as is?

Reply:
good point! we can do integration test by checking the existence of audit file for a specific word (such as the os username)